main.js

```diff
@@ -0,0 +1,22 @@
+// Hide Menu
+
+const closeButton = document.querySelectorAll('.close-menu');
+
+function hideMenu() {
+  document.querySelector('.app-menu').classList.replace('menu-show', 'menu-hide');
+}
+
+closeButton[0].addEventListener('click', hideMenu);
+closeButton[1].addEventListener('click', hideMenu);
+closeButton[2].addEventListener('click', hideMenu);
+closeButton[3].addEventListener('click', hideMenu);
```

Comment:
- [OPTIONAL] I highly recommend you to follow javascript best practices(KISS, YAGNI,  DRY) which will make your repo more professional :wink: 
   You can use the `forEach` method which allows you to iterate through the array without repeating yourself.
  #### _hint: `closeButton.forEach((btn) => btn.addEventListener('click', hideMenu));`_